NOTICES.txt

```diff
@@ -1,5 +1,5 @@
 NUnit 3.0 is based on earlier versions of NUnit, with Portions
```

Comment:
I didn't realize  that NUnit 3 used this NOTICES file that I created when transitioning from V2 to V3, in order to give credit to the past developers of NUnit V2. It doesn't really have anything to do with the old code, so I'd remove it.